deployment/terraform/container_service.tf

```diff
@@ -0,0 +1,205 @@
+locals {
+  app_image = "${module.ecr_repository.repository_url}:${var.image_tag}"
+}
+
+#
+# Security Group Resources
+#
+resource "aws_security_group" "alb" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgLoadBalancer"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_security_group" "ecs_service" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgEcsService"
```

Comment:
Similar comment here about the associated service. It looks like we associate it directly with the Django Fargate service, so adding something like `App` in the name helps avoid challenges in a future scenario where we need to add another ECS service that cannot share the security group.